collections/_infographics/policies/emissions-pricing-world.md

```diff
@@ -13,4 +13,56 @@ data-orig:
+* China has eight pilot emission trading systems in operation so far, with a national ETS to be launched in 2020. The coverage is based on preliminary unofficial estimates from an announcement by the State Development and Reform Commission of the People's Republic of China. The national ETS will initially cover only the power sector, with other sectors covered by individual pilot emission trading systems.
+
+## The underlying study of the infographic
+
+We took the infographic from a [study](https://openknowledge.worldbank.org/handle/10986/31755) published in 2019 by the Global Climate Change Group of the World Bank Group. **The study maps 57 initiatives for charging emissions at the state and regional levels and describes which countries choose which measures and how they implement them.** The most important conclusions of the study:
+
+* Greenhouse gas pricing initiatives cover **20% of global emissions**.
+* The number of initiatives is growing rapidly, with 38 implemented in the last 10 years.
+* 51% of covered emissions are below $10 per tonne CO2eq, while **the price that would have led to meeting the Paris Climate Agreement targets is $40-$80** per tonne CO2eq, depending on location, which in effect meets only 5% of covered emissions. Sweden has introduced the highest unit price at $127 per tonne  CO<sub>2</sub>eq.
+* The study also looks at so-called implicit emissions pricing, which includes, for example, higher taxation of fossil fuels. Therefore, the total share of emissions covered by any legislation is still slightly higher than the 20% mentioned above.
+
+The study is also summarized in a short [interview](https://www.youtube.com/watch?v=jBcpZahjKcE)
+with Celine Ramstein, one of the study's authors.
+
+## What does this infographic bring to the discussion?
+
+Emissions pricing measures are generally considered effective tools for achieving climate goals. However, without context and background, it is difficult to see how widespread these measures have become and which countries have not yet introduced any greenhouse gas pricing measures. The graphic can help to shift the debate away from arguments such as *"China needs to do something first"* to questions such as:
+
+* *How high is the price of emissions in each country?*
+* *What effect has the introduction of the measure had on the country's emissions and economy?*
+* *How can the introduction of measures in other countries be influenced?*
+* *Is it possible to link emissions trading schemes and introduce a single international price or carbon tax?*
+
+Financial instruments for mitigating climate change are discussed in more detail in the [International Monetary Fund study](https://www.imf.org/en/Publications/FM/Issues/2019/09/12/fiscal-monitor-october-2019)<!--, from which we have translated [infographic](/infographics/mitigacni-opatreni-mmf), which compares the different mechanisms for controlling greenhouse gas emissions-->.
```

Comment:
change from "in the" to "in an" (indefinite article)

Reply:
Corrected this. Thanks for catching this.